js/framework/widgets/eknWebview.js

```diff
@@ -113,8 +112,9 @@ var EknWebview = new Knowledge.Class({
         .then((model) => {
             if (model instanceof DModel.Article) {
                 let html = this.renderer.render(model);
-                let bytes = ByteArray.fromString(html).toGBytes();
-                let stream = Gio.MemoryInputStream.new_from_bytes(bytes);
+                let stream = Gio.MemoryInputStream.new_from_bytes(
```

Comment:
I think (but I'm not 100% sure) that you should be able to pass a Uint8Array to any function that expects a GBytes. So `ByteArray.fromString(html)` might work here.

Reply:
Ah! I was thinking that back and forth between types felt pretty awkward. Will try it. Thanks :)